### PR TITLE
SH-8: Create About us page

### PR DESCRIPTION
**Pull request by WeszeRita**
## JIRA ticket link:
https://shoperz.atlassian.net/browse/SH-8

### CODE REVIEW

File: src/app/pages/about-us/components/about-us/about-us.component.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<main class="text-grey-700 px-40 py-6">
+  <div class="pb-6">
+
```
Comment (krucsodaniel) [inline src/app/pages/about-us/components/about-us/about-us.component.html]: extra space

File: src/app/pages/about-us/components/about-us/about-us.component.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d997.1644710986066!2d26.049189365577227!3d44.481224171318786!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40b203ca74315c95%3A0x9070b3ecd57e3e9b!2sStrada%20Presei%2045%2C%20Bucure%C8%99ti%2C%20Romania!5e0!3m2!1sen!2shu!4v1695309396688!5m2!1sen!2shu"
+      class="w-full h-128"
+      loading="lazy"
+      referrerpolicy="no-referrer-when-downgrade">
+    </iframe>
+  </div>
+
+  <div class="flex flex-col md:flex-row gap-6">
+    <article class="bg-gradient-to-t from-lightTurquoiseForGradient to-darkTurquoiseForGradient rounded-lg text-justify p-6 md:p-10">
+
```
Comment (krucsodaniel) [inline src/app/pages/about-us/components/about-us/about-us.component.html:15]: extra space

File: src/app/pages/about-us/components/about-us/about-us.component.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      <h2 class="text-lg font-medium text-grey-900 py-6">{{ buildTranslationKey('aboutUs', 'about us') | translate }}</h2>
+      <p>{{ buildTranslationKey('aboutUs', 'about us text') | translate }}</p>
```
Comment (krucsodaniel) [inline src/app/pages/about-us/components/about-us/about-us.component.html]: can be `span` instead of `p`

File: src/app/pages/about-us/components/about-us/about-us.component.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+      <h2 class="text-lg font-medium text-grey-900 py-6">{{ buildTranslationKey('ourMission', 'our mission') | translate }}</h2>
+      <p>{{ buildTranslationKey('ourMission', 'our mission text') | translate }}</p>
```
Comment (krucsodaniel) [inline src/app/pages/about-us/components/about-us/about-us.component.html]: can be `span` instead of `p`

File: tailwind.config.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 const BASE_FONT_SIZE = 16;
-const pxToRem = (px) => `${ px / BASE_FONT_SIZE }rem`;
+const pxToRem = (px) => `${px / BASE_FONT_SIZE}rem`;
```
Comment (krucsodaniel) [inline tailwind.config.js]: Why did you remove the spaces in the template literal? Please put them back

File: src/assets/i18n/de-DE.json [hunk 6]
```diff
@@ -114,5 +114,19 @@
...
+    },
+    "ourMission": {
+      "our mission": "UNSERE AUFGABE",
+      "our mission text": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis praesentium vero voluptate! Asperiores dignissimos doloremque expedita explicabo impedit laudantium libero modi nisi quaerat sequi! Aliquid dolorem expedita inventore nostrum repellat repudiandae sit vero. Aliquam beatae culpa ea earum eius eos est expedita fugit harum, ipsum laborum modi nihil odio omnis perferendis quod ratione repellendus sit tempore, veritatis! Accusamus aliquam architecto aut deserunt dicta, dolores ducimus eligendi impedit incidunt ipsum iste labore libero maiores officia optio quam qui quidem quisquam reiciendis sed sequi soluta sunt temporibus ullam ut vero voluptatem voluptatum? Adipisci consectetur deserunt facilis necessitatibus nesciunt quaerat reprehenderit soluta veritatis."
+    },
+    "meetUs": {
+      "meet us": "UNS TREFFEN",
+      "presei square no45, bucharest - 099455": "Presei-Platz Nr. 45, 099455 Bukarest"
+    }
```
Comment (krucsodaniel) [inline src/assets/i18n/de-DE.json]: "aboutUs": {
      "about us": "ÜBER UNS",
      "about us text": "...."
    },
    
Instead maybe we could write something like this:

    "aboutUs": {
      "about us heading": "...",
      "about us text": "...."
    },

it is a bit confusing now, try to come up with clear naming for fields. same for "Mission" and "Meet us" part and in all 3 translation json files.

"presei square no45, bucharest - 099455" -> instead of this field name, we should just write address or something more simple, what do you think?

**Review by krucsodaniel — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 5 above)
Comment (WeszeRita) [reply tailwind.config.js]: Sorry, I used prettier

**Review by krucsodaniel — DISMISSED** [review]

File: src/app/app.module.ts [hunk 7]
```diff
@@ -7,20 +7,23 @@ import { CartModule } from './pages/cart/cart.module';
...
 import { ToastrModule } from 'ngx-toastr';
+import { AboutUsModule } from './pages/about-us/about-us.module';
+import { AboutUsComponent } from './pages/about-us';
 
 const routes: Routes = [
   { path: Route.base, redirectTo: Route.products, pathMatch: 'full' },
   { path: Route.products, component: ProductDashboardComponent },
   { path: Route.productById, component: ProductPageComponent },
   { path: Route.cart, component: CartComponent },
+  { path: Route.about, component: AboutUsComponent },
```
Comment (bodnya29179) [inline src/app/app.module.ts]: Try to use here Lazy Loading of AboutUsModule

File: src/app/pages/about-us/components/about-us/about-us.component.html [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+      src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d997.1644710986066!2d26.049189365577227!3d44.481224171318786!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40b203ca74315c95%3A0x9070b3ecd57e3e9b!2sStrada%20Presei%2045%2C%20Bucure%C8%99ti%2C%20Romania!5e0!3m2!1sen!2shu!4v1695309396688!5m2!1sen!2shu"
+      class="w-full h-128"
+      loading="lazy"
+      referrerpolicy="no-referrer-when-downgrade">
+    </iframe>
+  </div>
+
+  <div class="flex flex-col md:flex-row gap-6">
+    <article class="bg-gradient-to-t from-lightTurquoiseForGradient to-darkTurquoiseForGradient rounded-lg text-justify p-6 md:p-10">
+      <h2 class="text-lg font-medium text-grey-900 py-6">{{ buildTranslationKey('aboutUs', 'about us heading') | translate }}</h2>
```
Comment (bodnya29179) [inline src/app/pages/about-us/components/about-us/about-us.component.html]: don't use spaces when you're using json keys

Comment (bodnya29179) [reply src/app/pages/about-us/components/about-us/about-us.component.html]: Fix it in all places please

File: src/app/pages/about-us/components/about-us/about-us.component.html [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+      </div>
+
+      <div class="flex items-center mb-2">
+        <svg-icon class="mr-2" [src]="'email-icon'"></svg-icon>
+
+        <a class="block leading-10 text-grey-700" href="mailto:contact@shoperz.com" target="_top">contact@shoperz.com</a>
+      </div>
+
+      <div class="flex items-center mb-2">
+        <svg-icon class="mr-2" [src]="'phone-icon'" ></svg-icon>
```
Comment (bodnya29179) [inline src/app/pages/about-us/components/about-us/about-us.component.html]: extra space
```suggestion
        <svg-icon class="mr-2" [src]="'phone-icon'"></svg-icon>
```

**Review by bodnya29179 — CHANGES_REQUESTED** [review]

File: src/app/pages/about-us/components/about-us/about-us.component.html (hunk 8 above)
Comment (WeszeRita) [reply src/app/pages/about-us/components/about-us/about-us.component.html]: I made the same mistake with the footer. Can I fix it on this branch, or should I checkout to the footer branch and create a new mr?

Comment (bodnya29179) [reply src/app/pages/about-us/components/about-us/about-us.component.html]: Yes, you can

**Review by krucsodaniel — DISMISSED** [review]

**Review by krucsodaniel — APPROVED** [review]

**Review by bodnya29179 — APPROVED** [review]
